wrapanapi/systems/virtualcenter.py

```diff
@@ -544,6 +544,56 @@ def clone(self, vm_name, **kwargs):
         self.ensure_state(VmState.STOPPED)
         return self._clone(**kwargs)
 
+    @property
```

Comment:
I think these should be above any of the standard class methods, as they're properties+getters, in order to keep the class organized.